isamples_metadata/SESARTransformer.py

```diff
@@ -71,6 +71,15 @@ class MaterialCategoryMetaMapper(AbstractCategoryMetaMapper):
     _equalsBiogenicMapper = StringEqualityCategoryMapper(
         ["Macrobiology>Coral>Biology", "Coral>Biology"], "Biogenic non-organic material"
     )
+    _equalsNaturalSolidMapper = StringEqualityCategoryMapper(
```

Comment:
Adding new vocabulary. This is for the labels that are returned by the classification model. When we are doing rule-based classification, higher level of vocabulary is needed.